testing/features/support/components/footer/default.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Footer
+  class Default < Minimal
+    set_url "/iframe.html?id=components-footer--default-story&viewMode=story"
+
+    sections :category_titles, ".cads-grid-col-md-3 > h2" do
+      elements :links, :xpath, "./following-sibling::ul[1]/li/a"
+    end
+
+    element :website_feedback, ".cads-footer__feedback a"
```

Comment:
we need a validate method below this one. `has_category_titles?(wait: 5)` should suffice.